addons/tmf_loadouts/loadouts/chdkz.hpp

```diff
@@ -0,0 +1,510 @@
+	displayName = "Automatic Rifleman";
+	backpack[] = {"CUP_B_ACRScout_m95"};
+	primaryWeapon[] = {"CUP_arifle_RPK74"};
+	magazines[] = {
+		LIST_4("CUP_75Rnd_TE4_LRT4_Green_Tracer_762x39_RPK_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"B_FieldPack_khk"};
+	backpackItems[] = {
+		LIST_4("CUP_75Rnd_TE4_LRT4_Green_Tracer_762x39_RPK_M"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : car {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	headgear[] = {"H_Booniehat_oli"};
+	primaryWeapon[] = {"CUP_srifle_SVD"};
+	scope[] = {"cup_optic_pso_1_open"};
+	bipod[] = {"bipod_01_F_snd"};
+	magazines[] = {
+		LIST_11("CUP_10Rnd_762x54_SVD_M"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_MG"};
+	backpack[] = {"CUP_B_AlicePack_Khaki"};
+	primaryWeapon[] = {"CUP_lmg_PKM"};
+	magazines[] = {
+		LIST_5("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Yellow_M")
+	};
+};
+
+class mmgac : r {
+	displayName = "MMG Ammo Carrier";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Yellow_M")
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backpack[] = {"CUP_B_CivPack_WDL"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+	backpackItems[] = {
+		LIST_3("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Yellow_M"),
+		"ACE_EntrenchingTool"
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_DShkM_Gun_Bag"};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_DShkM_TripodHigh_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backPack[] = {"CUP_B_DShkM_TripodLow_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_RPG7V"};
+	secondaryAttachments[] = {"cup_optic_pgo7v2"};
+	magazines[] += {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M"),
+		LIST_3("CUP_OG7_M")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
```

Comment:
The Metis tripod bag is what contains the ammunition for the Metis static launcher.